### ISSUE TITLE
Unable to upload photos to blog 

### ISSUE THREAD

**Issue by srikumar101** [moxi9/phpfox#1335]
Hi, 

I am unable to upload photos to blog.. It says successfully uploaded but I just see this and not the path to the image like earlier: 

[img]<span class="no_image_item i_size_400"><span></span></span>[/img]

![image](https://cloud.githubusercontent.com/assets/10988234/14479425/f3d9ba26-013b-11e6-9ebf-a53c3baf5051.png)

Please help.

**Comment by neil96** [comment]
I can't reproduce this issue both on our demo and my localhost. Are you using cdn?

**Comment by srikumar101** [comment]
HI Neil,

I am able to upload it now. When I reverted back to earlier version for
some reason attachment folder was deleted, I added it manually, now its
working fine and storing the images in Amazon S3 bucket.

I see that old images (images which were uploaded before mid of march 2016
when Amazon S3 app was not updated) are still shown from EC2 file system
and not S3 bucket.

Older images location as seen in console - NOT pointing to S3 CDN (though
this image still exits in S3 and accessible though URL):

[image: Inline image 5]
New images location as seen in console - pointing to S3 bucket -

[image: Inline image 6]

Is this a bug?

Should I raise a separate ticket for this?

Regards,
Srini
